Kickstarter-iOS/Views/Cells/BackerDashboardProjectCell.swift

```diff
@@ -0,0 +1,87 @@
+import Foundation
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+internal final class BackerDashboardProjectCell: UITableViewCell, ValueCell {
+  fileprivate let viewModel: BackerDashboardProjectCellViewModelType = BackerDashboardProjectCellViewModel()
+
+  @IBOutlet fileprivate weak var cardView: UIView!
+  @IBOutlet fileprivate weak var mainContentContainerView: UIView!
+  @IBOutlet fileprivate weak var metadataBackgroundView: UIView!
+  @IBOutlet fileprivate weak var metadataIconImageView: UIImageView!
+  @IBOutlet fileprivate weak var metadataLabel: UILabel!
+  @IBOutlet fileprivate weak var percentFundedLabel: UILabel!
+  @IBOutlet fileprivate weak var projectNameLabel: UILabel!
+  @IBOutlet fileprivate weak var projectImageView: UIImageView!
+  @IBOutlet fileprivate weak var progressStaticView: UIView!
+  @IBOutlet fileprivate weak var progressBarView: UIView!
+  @IBOutlet fileprivate weak var savedIconImageView: UIImageView!
+
+  internal func configureWith(value: Project) {
+    self.viewModel.inputs.project(value)
```

Comment:
hmm whatcha mean, all our cells inherit from `ValueCell` and have to define `internal func configureWith(value: Value)`